modules/shortcodes/tweet.php

```diff
@@ -142,4 +143,61 @@ static public function jetpack_tweet_shortcode_script() {
 		}
 	}
 
+	static public function enqueue_block_editor_assets() {
+		wp_enqueue_script( 'wp-blocks' );
+		wp_enqueue_script( 'wp-i18n' );
+		wp_enqueue_script( 'wp-element' );
+		wp_enqueue_script( 'shortcode' );
+		add_action( 'admin_print_footer_scripts', array( __CLASS__, 'admin_footer' ), 999 );
+	}
+
+	static public function admin_footer() {
+?>
+<script>
+( function( wp ) {
+	var blockStyle = {
+			backgroundColor: '#900',
+			color: '#fff',
+			padding: '20px'
+		};
+
+	wp.blocks.registerBlockType( 'jetpack/tweet', {
+		title: wp.i18n.__( 'Tweet', 'jetpack' ),
+		icon: 'twitter',
+		category: 'layout',
+
+		attributes : {
+			tweet : wp.blocks.query.query( 'input[name=tweet]').value
+		},
+
+		edit : function( props ) {
+			return wp.element.createElement(
+				'input',
+				{
+					name : 'tweet',
+					type : 'url',
+					value : props.tweet
```

Comment:
`props` isn't the attributes of the block, but attributes can be accessed from `props.attributes`. There are a number of other values in props as well, such as `setAttributes` noted in the previous comment.

Instead, this would be: `props.attributes.tweet`